shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2885,8 +2886,7 @@ void WebContents::StartDrag(const gin_helper::Dictionary& item,
     base::CurrentThread::ScopedNestableTaskAllower allow;
     DragFileItems(files, icon->image(), web_contents()->GetNativeView());
   } else {
-    gin_helper::ErrorThrower(args->isolate())
-        .ThrowError("Must specify either 'file' or 'files' option");
+    thrower.ThrowError("Must specify either 'file' or 'files' option");
```

Comment:
nit: I prefer the previous code of constructing the thrower inline. Throwers have no state and it's not used anywhere else in this function.